person_class.rb

```diff
@@ -1,8 +1,9 @@
 require './corrector'
+require_relative 'rental'
 # Person class
 class Person
   attr_reader :id
-  attr_accessor :name, :age, :parent_permission
+  attr_accessor :name, :age, :parent_permission, :rental
 
   def initialize(age, parent_permission = 'true', name = 'Unknown')
     @id = Random.rand(1..500)
```

Comment:
- [ ] To fulfill the `has_many` relationship between Person and Rental, remember that a person can have many rentals. Let's add `rentals` to the initialized method. Add this `@rentals = []` to the `initialize` method. This is also why I suggested that you add `rentals` to the `attr_accessor` 😉.